Cookbook/Technical-Documents/ApplicationDistribution.md

```diff
@@ -0,0 +1,24 @@
+# Application Distribution
+This document is for any Babylon iOS developers who wish to distribute our applications, or for any partners who wish to gain access to our applications.
+
+## How to gain access to iOS applications
+### External Babylon partners
+* If you are an external partner of Babylon, please speak with your Babylon contact who will arrange this for you
+
+### Babylon Employees
+* If you are a Babylon employee, please reach out to the current iOS support engineer(s) in the #ios slack channel
+* You will also need to have [App Center](https://github.com/babylonhealth/ios-playbook/blob/simon/cnsmr-3226/Cookbook/Technical-Documents/AppCenter.md) installed
+
+## Distribution of iOS applications
+### Distributing to a Babylon employee
```

Comment:
```suggestion
### Distributing to AppCenter
```

Reply:
I'm going to rename this to `Distributing via App Center`